src/NServiceBus.Core/DelayedDelivery/TimeoutManager/Core/DefaultTimeoutManager.cs

```diff
@@ -18,7 +19,7 @@ public void PushTimeout(TimeoutData timeout)
         {
             if (timeout.Time.AddSeconds(-1) <= DateTime.UtcNow)
             {
-                var sendOptions = new DispatchOptions(timeout.Destination,new AtomicWithReceiveOperation(), new List<DeliveryConstraint>());
+                var sendOptions = new DispatchOptions(timeout.Destination,new AtomicWithReceiveOperation(), new List<DeliveryConstraint>(), new ContextBag());
```

Comment:
We should flow the context from the incoming `TimeoutMessageProcessorBehavior` I'll create a separate pull for that